tests/build_/test_build_request.py

```diff
@@ -1058,9 +1058,37 @@ def test_render_optional_plugins(self, tmpdir):
         (None, None, DEFAULT_BUILD_IMAGE),
         ('fedora:latest', 'buildroot-stream:v1.0', KeyError)
     ))
+    @pytest.mark.parametrize('additional_kwargs', (
+        {
+            'authoritative_registry': 'authoritative_registry',
+            'distribution_scope': 'distribution_scope',
+            'info_url_format': 'info_url_format',
+            'kojihub': 'kojihub',
+            'kojiroot': 'kojiroot',
+            'pulp_registry': 'pulp_registry',
+            'registry_api_versions': ['v1', 'v2'],
+            'smtp_additional_addresses': ['spam@food.bz', 'bacon@food.bz'],
+            'smtp_email_domain': 'smtp_email_domain',
+            'smtp_error_addresses': ['error1@foo.com', 'error2@foo.com'],
+            'smtp_from': 'smtp_from',
+            'smtp_host': 'smtp_host',
+            'smtp_to_pkgowner': 'smtp_to_pkgowner',
+            'smtp_to_submitter': 'smtp_to_submitter',
```

Comment:
These two lines should be e.g. True and False, to test that Booleans get passed through correctly.